qemu/qemu_test.go

```diff
@@ -236,3 +236,25 @@ func TestAppendQMPSocket(t *testing.T) {
 
 	testAppend(qmp, qmpSocketString, t)
 }
+
+var qemuString = "-name cc-qemu -cpu host -uuid 123456789"
+
+func TestAppendStrings(t *testing.T) {
+	var params []string
+
+	config := Config{
+		Path:     "qemu",
+		Name:     "cc-qemu",
+		UUID:     "123456789",
```

Comment:
Yep, as per my overall comment on the patch, we need to decide the right place(s?) to do meaningful validation.